tests/sources/tools/utils/test_hyperparameter_tuner.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from optuna.study.study import Study
+from opendr.utils.hyperparameter_tuner.dummy_learner import DummyLearner
+from opendr.utils.hyperparameter_tuner.hyperparameter_tuner import HyperparameterTuner
```

Comment:
Can we simplify this import statement by directly exposing the interface in ``__init__.py`` similar to the rest of the tools  #133  as:

```python
from opendr.utils.hyperparameter_tuner import HyperparameterTuner
```

Reply:
Thank you Nikos! That is a good idea, I have add HyperparameterTuner to the __init__.py